elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -365,28 +342,11 @@ public void bindEntity(Class<?> cls) {
             log.error("Duplicate binding {} for {}, {}", type, cls, duplicate);
             throw new DuplicateMappingException(type.toString() + " " + cls.getName() + ":" + duplicate.getName());
         }
-        bindEntityToJsonApi.put(cls, type);
+
+        entityBindings.putIfAbsent(lookupEntityClass(cls), new EntityBinding(cls, type, databaseManager));
```

Comment:
Should check for duplicates. Fail iff `cls` exists and `databaseManager` is the same. This way if a DB manager (i.e. hibernate3) sends us the same mapping multiple times, we ignore it. However, if you've mapped this bean twice you then get a very loud notice that this occurred.